style/pi-hole.css

```
@@ -330,6 +330,10 @@
   background: #367fa9;
 }
 
+.sidebar-menu-item {
+  padding-left: 5px;
+}
+
```

```suggestion
.sidebar-menu .fa-fw {
  margin-right: 5px;
}
```

and remove all changes from `header.php`